renku_notebooks/util/_gitlab.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integrating interactive environments with GitLab."""
+
```

Comment:
Is there a reason that you used `_gitlab.py`? This file along with `_jupyterhub.py` and `_kubernetes.py` are used outside this module and should have public name IMO.

Reply:
I wanted to avoid confusion with python libraries which are also called `gitlab` and `kubernetes` and `jupyterhub` is a package -- hence the `_`. But I'm open to other naming suggestions.

Reply:
OK, I see. No better suggestion other than putting _ at the end. But really it is not that important and can also be changed later if we come up with a better solution.

Reply:
ok, I like the `_` at the end - doesn't make it look like these are supposed to be private.